core/src/apps/cardano/helpers/__init__.py

```diff
@@ -1,7 +1,8 @@
 from trezor import wire
 
 INVALID_ADDRESS = wire.ProcessError("Invalid address")
-NETWORK_MISMATCH = wire.ProcessError("Output address network mismatch!")
+INVALID_ADDRESS_PARAMETERS = wire.ProcessError("Invalid address parameters")
+NETWORK_MISMATCH = wire.ProcessError("Output address network mismatch")
```

Comment:
I think this error is relevant for Catalyst as well in which case I'd reword the message to "Address network mismatch"

Reply:
I realized address params don't contain the network id/protocol magic so this is apparently relevant only for output addresses given in their raw form